janusgraph-berkeleyje/src/test/java/org/janusgraph/BerkeleyStorageSetup.java

```diff
@@ -14,12 +14,30 @@
 
 package org.janusgraph;
 
+import org.janusgraph.diskstorage.berkeleyje.BerkeleyJEKeyValueStore;
 import org.janusgraph.diskstorage.configuration.ModifiableConfiguration;
 import org.janusgraph.diskstorage.configuration.WriteConfiguration;
 
 import static org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration.*;
 
+import java.time.Instant;
+
+import com.sleepycat.je.dbi.TTL;
+import com.sleepycat.je.utilint.TestHookAdapter;
+
 public class BerkeleyStorageSetup extends StorageSetup {
+    static {
+        BerkeleyJEKeyValueStore.hoursIsSeconds = true;
+        TTL.setTimeTestHook(new TestHookAdapter<Long>() {
+            private static final long SECONDS_PER_HOUR = 60 * 60;
+            private final long initial = Instant.now().toEpochMilli();
+
+            @Override
+            public synchronized Long getHookValue() {
+                return initial + (Instant.now().toEpochMilli() - initial) * SECONDS_PER_HOUR;
```

Comment:
`(Instant.now().toEpochMilli() - initial)` computes the time in milliseconds passed since `initial` and this is multiplied with the number of `SECONDS_PER_HOUR`. Shouldn't this be multiplied with the number of milliseconds per hour?

Reply:
We need speed up our clock, those when 1 second passed bdb should think 1 hour passed, so multiplier is 3600. If we use 3600 * 1000, bdb will think 1000 hours passed. It needs because berkeleydb support only hour discrete TTL